src/applications/disability-benefits/all-claims/pages/trainingPayWaiver.js

```diff
@@ -0,0 +1,29 @@
+import fullSchema from '../config/schema';
+import { waiveTrainingPayDescription } from '../content/trainingPayWaiver';
+
+const { waiveTrainingPay } = fullSchema.properties;
+
+export const uiSchema = {
+  'ui:title': 'Training pay waiver',
+  waiveTrainingPay: {
+    'ui:title': ' ',
+    'ui:description': waiveTrainingPayDescription,
+    'ui:widget': 'yesNo',
+    'ui:options': {
+      yesNoReverse: true,
```

Comment:
Oh hey. Didn't know this was a thing. What's it do exactly?

Reply:
Yeah, this is awful. It literally reverses answers, so `true` would make its way into the data as `false`. The reason for this here is that the question is worded in such a way that 'yes means no' and vice-versa. Given:
```

Reply:
:thinking: So if they select `I want to get...`, it'll get sent up to the api as `waiveTraningPay: true`. That makes sense.

Could we just reverse the question labels, though?

```js
labels: {
  N: 'I don’t want to get VA compensation pay for the days I receive training pay.',
  Y: 'I want to get VA compensation pay instead of training pay.',
},
```

As far as I'm concerned, it's six of one, half-dozen of the other, but I'm curious.